game_cfg/colors.h

```diff
@@ -1,7 +1,5 @@
-#include <ncurses.h>
-
+// Text colors
 #define DEFAULT -1
-
 #define BLACK 0
```

Comment:
Making the colors an enum would be neat. That way you could use them in a type safe way, I think.